telegram/ext/_jobqueue.py

```diff
@@ -163,8 +163,8 @@ async def callback(context: CallbackContext)
                 be available in the callback.
 
                 .. versionadded:: 20.0
-            context (:obj:`object`, optional): Additional data needed for the callback function.
-                Can be accessed through :attr:`Job.context` in the callback. Defaults to
+            data (:obj:`object`, optional): Additional data needed for the callback function.
+                Can be accessed through :attr:`Job.data` in the callback. Defaults to
                 :obj:`None`.
```

Comment:
```suggestion
                Can be accessed through :attr:`Job.data` in the callback. Defaults to
                :obj:`None`.
                
                .. versionchanged:: 20.0
                    Renamed the parameter ``context`` to :paramref:`data`.
```
Same for all the other run_* methods